intellij-setup.md

```diff
@@ -28,23 +28,27 @@ Android Studio (and various other JetBrains editors) is currently not supported.
 
 ### Installing the plugins
 
-We provide two plugins:
+To use IntelliJ with Flutter, you must install two plugins:
 
   * The `Dart` plugin offers code analysis (code validation as you type, code completions, etc.).
   * The `Flutter` plugin powers Flutter developer workflows (running, debugging, hot reload, etc.).
 
 To install the plugins:
 
-1. Open plugin preferences (**Preferences>Plugins** on macOS, **File>Settings>Plugins** on Linux)
-1. Select **"Browse repositories…"**
-1. Search for `'dart'`, select the Dart plug-in and click `install` (do not restart yet).
+1. Start IntelliJ.
+1. Open plugin preferences (**Preferences>Plugins** on macOS, **File>Settings>Plugins** on Linux).
```

Comment:
How easy would it be to do an OS detect + 2 versions of the page? That might make it a bit easier to read. We used to do this for Dart I think - and we'll almost certainly need to do it for Windows.

Perhaps something for the future?